drivers/rilmodem/call-barring.c

```diff
@@ -33,6 +33,7 @@
 #include "call-barring.h"
 #include "rilmodem.h"
 #include "ril_constants.h"
+#include "rilutil.h"
```

Comment:
I removed it from rilmodem.h and moved to the atoms because the definitions are not actually used in rilmodem.h and because rilmodem.h is included in ril plugins files (like mtk2.c) that do not need rilutil.h too.